STARTERKIT/webpack.config.js

```diff
@@ -0,0 +1,29 @@
+var webpack = require('webpack');
+
+var config = {
+  context: __dirname + '/js',
+  entry: {
+    app: './init.js',
+  },
+  output: {
+    path: __dirname + '/dist',
+    filename: 'app.js',
+  },
+  module: {
+    loaders: [
+      {
+        test: /\.js$/,
+        loader: 'babel-loader',
+        query: {
+          presets: ['es2015']
+        }
+      }
+    ]
+  },
+  stats: {
+    colors: true
+  },
+  devtool: 'source-map'
+};
+
```

Comment:
After #5 gets merged, we need to handle the `NODE_ENV` variable here when it gets the value of `production` and add uglifyjs as a webpack plugin,

something like:
```js
if (process.env.NODE_ENV === 'production') {
    config.plugins.push(
        new webpack.optimize.UglifyJsPlugin({
            compress: {
                screw_ie8: true
            }
        })
    )
}
```